### DISCUSSION TITLE
关于VLESS-xtls-rprx-vision + quic配置

### DISCUSSION THREAD

**Discussion by chise0713** [XTLS/Xray-core#2193] [Q&A] [answered]
假设我配置VLESS-xtls-rprx-vision + quic我还需要再套一层加密吗？如REALITY和TLS。
单用quic自带的加密够安全吗
看文档说必须开启TLS，是在security项内设置tls就行了吗？
若是我所在的区域有sni白名单的话该怎么办？

**Comment by flowerinsnowdh** [comment, marked as answer]
https://xtls.github.io/config/transports/quic.html#quicobject

> **警告**
> 对接的两端的配置必须完全一致，否则连接失败。 QUIC 强制要求开启 TLS，在传输配置中没有开启 TLS 时，Xray 会自行签发一个证书进行TLS 通讯。

QUIC 底层传输协议必须开启 TLS ，无论是主动地开启还是程序自动开启，它的传输层加密都是 TLS

# 至于REALITY
> REALITY only supports TCP, H2, gRPC and DomainSocket for now.

REALITY 不支持 QUIC 作为底层传输协议

**Reply by FranzKafkaYu** [reply]
xtls-rprx-vision is tcp only.

**Reply by flowerinsnowdh** [reply]
> xtls-rprx-vision is tcp only.

感谢指出

@KoinuDayo

xtls-rprx-vision 只支持 tcp
